android/src/main/res/layout/fingerprint_dialog.xml

```diff
@@ -1,79 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/fingerprint_container"
-    android:layout_width="match_parent"
+    android:layout_width="250dp"
+    android:bottomLeftRadius="30dp"
+    android:bottomRightRadius="30dp"
+    android:topLeftRadius="30dp"
+    android:topRightRadius="10dp"
     android:layout_height="match_parent"
+    android:paddingTop="15dp"
+    android:paddingBottom="15dp"
+    android:paddingLeft="30dp"
+    android:paddingRight="30dp"
```

Comment:
these should all be in dimens files

Reply:
sure i will update. Thanks @rolandolopez